emukungu/IOT
# set up user authentication

#### What does this PR do?
- Add user authentication

#### Description of Task to be completed?
-Sign out, login, logout a user

#### How should this be manually tested?
1. Check out to the branch `ft-authentication`
2. Create a virtual environment for the project, run `virtual env`
3. Install dependencies using `pip install -r requirements.txt`
4. On terminal, run command `python manage.py runserver`

#### Any background context you want to provide?
- N/A

#### What are the relevant pivotal tracker stories?
N/A

#### Postman Documentation
- N/A

#### Screenshots (If applicable)
N/A
